src/OSPSuite.Presentation/Binders/ObservedDataDragDropBinder.cs

```diff
@@ -24,6 +25,25 @@ public IReadOnlyList<DataRepository> DroppedObservedDataFrom(IDragEvent e)
          return getObservedDataNodesFrom(e).Select(observedDataNode => observedDataNode.Tag.Subject).ToList();
       }
 
+      public Cache<string, List<DataRepository>> DroppedObservedDataWithFolderPathFrom(IDragEvent e)
+      {
+         var dataNodes = e.Data<IEnumerable<ITreeNode>>();
+
+         if (dataNodes == null)
+            return new Cache<string, List<DataRepository>>();
+
+         var treeNodes = dataNodes as IList<ITreeNode> ?? dataNodes.ToList();
+
+         if (areAllObservedDataNodes(treeNodes))
```

Comment:
Can you add a comment here? Something like "This is the case when dropping one or more observed data from multi select" etc..

Reply:
comment added for both cases